lol_alloc/src/free_list_allocator.rs

```diff
@@ -173,14 +173,13 @@ fn full_size(layout: Layout) -> usize {
 // From https://github.com/wackywendell/basicalloc/blob/0ad35d6308f70996f5a29b75381917f4cbfd9aef/src/allocators.rs
 // Round up value to the nearest multiple of increment
```

Comment:
It would be good to document the new requirement on increment here.

And while you are at it, a doc comment on multiple_below for it as well would be good.